core/actor/src/main/scala/net/liftweb/actor/LAPinger.scala

```diff
@@ -34,8 +34,8 @@ object ThreadPoolRules {
 }
 
 /**
- * The ActorPing object schedules an actor to be ping-ed with a given message at specific intervals.
- * The schedule methods return a ScheduledFuture object which can be cancelled if necessary
+ * The LAPinger object schedules a LiftActor to be pinged "delay" miliseconds in the future.
```

Comment:
The original phrasing seems clearer to me here (class names aside). `delay` doesn't really have a meaning at the level of the singleton that I can see.